test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -744,6 +744,31 @@ ConstructorTestClass(int i, bool b)
         $res.CompletionMatches[0].CompletionText | Should -BeExactly '$TestVar1'
     }
 
+    Context 'Scope parameter completion' {
+        BeforeAll {
+            $allScopes = 'Global', 'Local', 'Script'
+            $globalScope = 'Global'
+            $localScope = 'Local'
+            $scriptScope = 'Script'
+            $allScopeCommands = 'Clear-Variable', 'Export-Alias', 'Get-Alias', 'Get-PSDrive', 'Get-Variable', 'Import-Alias', 'New-Alias', 'New-PSDrive', 'New-Variable', 'Remove-Alias', 'Remove-PSDrive', 'Remove-Variable', 'Set-Alias', 'Set-Variable'
+        }
+
+        It "Should complete '<ParameterInput>' for '<Commands>'" -TestCases @(
+            @{ Commands = $allScopeCommands; ParameterInput = "-Scope "; ExpectedScopes = $allScopes }
+            @{ Commands = $allScopeCommands; ParameterInput = "-Scope G"; ExpectedScopes = $globalScope }
+            @{ Commands = $allScopeCommands; ParameterInput = "-Scope Lo"; ExpectedScopes = $localScope }
+            @{ Commands = $allScopeCommands; ParameterInput = "-Scope Scr"; ExpectedScopes = $scriptScope }
+            @{ Commands = $allScopeCommands; ParameterInput = "-Scope NonExistentScope"; ExpectedScopes = @() }
```

Comment:
Why `@()`, not empty string? Could you please clarify?

Reply:
There are another examples such pattern but there Join is used. So I think we should explicitly use empty string.

Reply:
@iSazonov Fixed with checking against strings in a865a9dc3